### DISCUSSION TITLE
The UI cannot be opened in ssh remote mode

### DISCUSSION THREAD

**Discussion by YeZengHui** [slint-ui/slint#3758] [Q&A]
The UI cannot be opened in ssh remote mode

**Comment by hunger** [comment]
Thank you for taking the time to report your issue with Slint.

Some more information would be helpful to figure out what is happening:

* How exactly do you use ssh to connect to the remote system?
* What kind of UI is running on that local and the remote system (wayland or X11)?
* Which rendering backend does your client use?

**Reply by tronical** [reply]
I think @YeZengHui reported this also in https://github.com/slint-ui/slint/issues/3757
